.gitattributes

```diff
@@ -142,3 +142,4 @@ cmd/workspace/workspace/workspace.go linguist-generated=true
 bundle/internal/tf/schema/\*.go linguist-generated=true
 go.sum linguist-generated=true
 bundle/schema/jsonschema.json linguist-generated=true
+acceptance/** eol=lf
```

Comment:
Does it make sense to split this file? Move generated .gitattributes into cmd/.gitattributes and keep manually maintained ones at root.

Also, should this setting apply only to acceptance tests or to the whole repo?

Reply:
> should this setting apply only to acceptance tests or to the whole repo?

Only acceptance tests. No reason I can think of upfront to apply to the whole repo. That could have adverse consequences of checking out the repo in windows.